built_value/lib/standard_json_plugin.dart

```diff
@@ -50,6 +50,11 @@ class StandardJsonPlugin implements SerializerPlugin {
       } else {
         return _toMap(object, _needsEncodedKeys(specifiedType));
       }
+    } else if(object is List<dynamic>) {
```

Comment:
This matches too much; 'is List<dynamic>' returns true for every list.

I think instead you'll want to check if object.runtimeType == <dynamic>[].runtimeType. It probably makes sense to cache <dynamic>[].runtimeType in a static field.

Reply:
Done. I've also noticed that I have patched a wrong place (afterSerialize instead of beforeSerialize), fixed